bug(commands): scaffold emits stray `pub mod ws_urls;` at app module root causing E0583

## Summary

When generating an app with `reinhardt-admin startapp <name> --with-pages` (or the equivalent scaffold), the generated `src/apps/<name>.rs` contains a top-level `pub mod ws_urls;` declaration that has no corresponding file. This causes an immediate `E0583` compile error.

## Error

```
error[E0583]: file not found for module `ws_urls`
  --> src/apps/posts.rs:13:1
   |
13 | pub mod ws_urls;
   | ^^^^^^^^^^^^^^^^
   |
   = help: to create the module `ws_urls`, create file "src/apps/posts/ws_urls.rs" or "src/apps/posts/ws_urls/mod.rs"
```

## Root Cause

`ws_urls` is already declared inside `src/apps/<name>/urls.rs`:

```rust
// src/apps/<name>/urls.rs
#[cfg(server)]
pub mod ws_urls;  // ← correct location
```

The scaffold also emits a redundant top-level declaration in `src/apps/<name>.rs`:

```rust
pub mod urls;
pub mod views;
pub mod ws_urls;  // ← BUG: stray, file does not exist at this path
```

The compiler looks for `src/apps/<name>/ws_urls.rs` (which does not exist), while the actual file is `src/apps/<name>/urls/ws_urls.rs`.

## Affected Templates

- `app_pages_template` (and any template that generates a `ws_urls` submodule)

## Expected Behavior

The top-level `pub mod ws_urls;` line should be removed from the app module root template. Only `pub mod urls;` is needed there; `ws_urls` is a submodule of `urls`.

## Proposed Fix

In the relevant `app_*_template` file (e.g. `crates/reinhardt-commands/templates/app_pages_template/src/apps/app_name.rs.tpl`), remove:

```rust
pub mod ws_urls;
```

## Workaround

Manually remove the stray `pub mod ws_urls;` line from `src/apps/<name>.rs` after scaffolding.

## Environment

- `reinhardt-admin` version: `0.1.0-rc.18` / `0.1.0-rc.19`
- Rust edition: 2024

--- a/crates/reinhardt-commands/templates/app_pages_template/lib.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_template/lib.rs.tpl
@@ -10,7 +10,6 @@ pub mod models;
 pub mod serializers;
 pub mod urls;
 pub mod views;
-pub mod ws_urls;
 
 // Client-side modules
 #[cfg(client)]

--- a/crates/reinhardt-commands/templates/app_pages_workspace_template/lib.rs.tpl
+++ b/crates/reinhardt-commands/templates/app_pages_workspace_template/lib.rs.tpl
@@ -10,7 +10,6 @@ pub mod models;
 pub mod serializers;
 pub mod urls;
 pub mod views;
-pub mod ws_urls;
 
 // Client-side modules
 #[cfg(client)]

